libs/ddhub-client-gateway-message-broker/src/lib/services/ddhub-base.service.ts

```diff
@@ -151,7 +151,7 @@ export abstract class DdhubBaseService {
     }
 
     throw new MessageBrokerException(
-      e.message,
+      `${e.message} -> ${e.response.data.returnMessage}`,
```

Comment:
What is the difference between `e.message` and `e.response.data.returnMessage`?

Reply:
e.message from CG , e.response.data.returnMessage from MB

Reply:
add it as additional property in the 
```
libs/ddhub-client-gateway-message-broker/src/lib/exceptions/message-broker.exception.ts
```